cli/dcoscli/node/main.py

```diff
@@ -532,7 +574,9 @@ def _dcos_log(follow, lines, leader, slave):
     if not dcos_url:
         raise config.missing_config_exception(['core.dcos_url'])
 
-    url = dcos_url + endpoint + endpoint_type + '/?skip_prev={}'.format(lines)
+    url = (dcos_url + endpoint + endpoint_type +
```

Comment:
we need some url cleansing here (ie be careful of which variable has what slash.

dcos_url.rstrip('/') + endpoint...

Maybe add a quick unit test with parameters with and without trailing slashes to make sure theres no bug here